Sources/SwiftDEBot/Utils/Client.swift

```diff
@@ -0,0 +1,30 @@
+import DiscordBM
+
+extension DiscordClient {
+    @discardableResult
+    func send(_ message: String, to channelID: String) async throws -> DiscordClientResponse<DiscordChannel.Message> {
+        try await self.createMessage(channelId: channelID, payload: .init(content: message))
+    }
+
+    @discardableResult
+    func addReaction(_ emoji: Reaction, to messageID: String, in channelID: String) async throws -> DiscordHTTPResponse {
```

Comment:
Oh, good to know! Thanks for the well-maintained library. It's so nice coming from an old archived package that went so far as forcing me to manually update CA certs in the docker container because they had expired 😅